vscode-dotnet-runtime-library/src/Acquisition/InstallScriptAcquisitionWorker.ts

```diff
@@ -35,6 +39,14 @@ export class InstallScriptAcquisitionWorker implements IInstallScriptAcquisition
             return this.scriptFilePath;
         } catch (error) {
             this.eventStream.post(new DotnetInstallScriptAcquisitionError(error));
+
+            // Try to use fallback install script
+            const fallbackPath = this.getFallbackScriptPath();
+            if (fs.existsSync(fallbackPath)) {

```

Comment:
Just for clarity, this wouldn't exist if the extension failed to install right?

Reply:
Yes, if it failed to install, if something went wrong with packaging and the install scripts didn't get included, or in tests.